src/app/pages/subjects/subjects.js

```diff
@@ -0,0 +1,149 @@
+import BaseView from '~/helpers/backbone/view';
+import BaseModel from '~/helpers/backbone/model';
+import {props} from '~/helpers/backbone/decorators';
+import {template} from './subjects.hbs';
+import FilterButton from './filter-button/filter-button';
+import CategorySection from './category-section/category-section';
+
+function makeBook(title, categories) {
+    return {
+        title,
+        categories,
+        coverUrl: `https://placeholdit.imgix.net/~text?txtsize=33&txt=${title}&w=200&h=180`
+    };
+}
+
+@props({
+    template: template,
+    regions: {
+        filterButtons: '.filter-buttons',
+        bookViewer: '.book-viewer'
+    }
+})
+export default class Subjects extends BaseView {
+
+    constructor() {
+        super();
+        this.model = new BaseModel({viewIsReady: false});
+
+        let updateBookIndexInfo = () => {
+            let setHtmlToMember = (nodeName, memberName) => {
+                this.el.querySelector(`[data-manager="${nodeName}"]`)
+                .innerHTML = this.model.get('bookIndex')[memberName];
+            };
+
+            if (this.model.get('viewIsReady') &&
+                this.model.has('bookIndex')) {
+                setHtmlToMember('page-description', 'page_description');
+                setHtmlToMember('ds1-head', 'dev_standard_1_heading');
+                setHtmlToMember('ds1-body', 'dev_standard_1_description');
+                setHtmlToMember('ds2-head', 'dev_standard_2_heading');
+                setHtmlToMember('ds2-body', 'dev_standard_2_description');
+                setHtmlToMember('ds3-head', 'dev_standard_3_heading');
+                setHtmlToMember('ds3-body', 'dev_standard_3_description');
```

Comment:
This function seems to be fighting with the templating engine.  Why not let it handle updating DOM elements?